Makefile.am

```diff
@@ -161,8 +161,9 @@ gen_%.o: src/gen_%.c
 $(gen_context_BIN): $(gen_context_OBJECTS)
 	$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $(LDFLAGS_FOR_BUILD) $^ -o $@
 
+ECC_SECP=libecc_secp256k1.a
```

Comment:
nit: `ECC` does not say much, easily confused. Maybe `NAIVE` or similar?

Reply:
fixed.